src/Symfony/Component/Process/Messenger/ProcessMessage.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Messenger;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class ProcessMessage implements \Stringable
+{
+    public function __construct(
+        public readonly string $command,
+        public readonly ?string $cwd = null,
+        public readonly ?array $env = null,
+        public readonly mixed $input = null,
+        public readonly ?float $timeout = 60,
+    ) {
+    }
+
+    public function __toString(): string
```

Comment:
generally what about just dumping payloads $dump($messsage) in CLI for -vvv?

Reply:
Can you explain this a bit more, I'm not understanding what you mean.

Reply:
https://github.com/symfony/console/blob/aa5d64ad3f63f2e48964fc81ee45cb318a723898/Helper/Dumper.php#L22

`$dump = new Dumper($output);`